docs/reference/components/sources/http.cue

```diff
@@ -83,6 +83,33 @@ components: sources: http: {
 				items: type: string: examples: ["User-Agent", "X-My-Custom-Header"]
 			}
 		}
+		auth: {
```

Comment:
This auth fragment applies to logplex as well yes?

Reply:
Yes! In logplex:
```
auth:    sources.http.configuration.auth
```